commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleMotionProfileControllers.scala

```diff
@@ -0,0 +1,89 @@
+package com.lynbrookrobotics.potassium.commons.drivetrain
+
+import java.lang.Math._
+
+import com.lynbrookrobotics.potassium.{PeriodicSignal, Signal}
+import squants.motion._
+
+trait UnicycleMotionProfileControllers[DriveSignal, DrivetrainHardware <: UnicycleHardware, DrivetrainProperties <: UnicycleProperties]
+  extends UnicycleCoreControllers[DriveSignal, DrivetrainHardware, DrivetrainProperties] {
+
+  /**
+    * This controller uses a control loop that results in constant acceleration,
+    * then a constant cruising velocity, and then a constant deceleration to the
+    *  target final velocity at the target position. It is meant to control
+    *  second order systems.
+    * @param initVelocity in feetPerSecond
```

Comment:
Because we no longer care about the actual value units (we only care that they are the correct quantity type), can you remove the annotations about the units of each param?